doc/install.rst

```diff
@@ -0,0 +1,83 @@
+Build Instructions
+==================
+
+Dependencies
+------------
+
+We are using colcon_ as build tool and typically use `ros2 run` to execute our
+applications.  While we are not really depending on any ROS_ packages, this
+means a basic ROS 2 installation is recommended.
+
+We are testing on Ubuntu 20.04 with ROS Foxy.  Other versions may work as well
+but are not officially supported.
+
+
+Get the Source
+--------------
+
+**blmc_drivers** depends on several other of our packages which are
+organized in separate repositories.  We therefore use a workspace management
+tool called treep_ which allows easy cloning of multi-repository projects.
+
+treep can be installed via pip::
+
+    pip install treep
+
+Clone the treep configuration containing the "BLMC_DRIVERS" project::
+
+    git clone git@github.com:machines-in-motion/treep_machines_in_motion.git
+
+**Note:**  treep searches for a configuration directory from the current working
+directory upwards.  So you can use treep in the directory in which you invoked
+the ``git clone`` command above or any subdirectory.
+
+Now clone the project (including packages for communication with a master
+board)::
+
+    treep --clone BLMC_DRIVERS
+
+or, if you are only interested in CAN communication::
+
+    treep --clone BLMC_DRIVERS_CAN_ONLY
```

Comment:
This is a separate discussion not related to documentation but do you think it would be possible to separate the drivers for the ethernet/master-board and the CAN communication?  Something like blmc_drivers_can and blmc_drivers_eth?

Right now, I have master-board-related packages in my workspace because blmc_drivers depends on it, despite me not using them.

Reply:
We don't rely on the blmc_drivers for the eth anymore. We use the odri_control_interface for all our robots by now, which talks to the master-board sdk directly.

So if you want, we can drop the eth support in the blmc_drivers package completely and then only pull the CAN related code (so no master-board etc) when pulling the blmc_drivers package.

What do you think?